### ISSUE TITLE
save files missing

### ISSUE THREAD

**Issue by egoodchild** [wolveix/satisfactory-server#225]
**Describe the Bug**
My save files disappeared.

Been playing on the server for ~3 weeks, loaded up today to find server saying it was unclaimed.  Ruh roh.  Restarted it, same.  

Looked into the docker folder as listed in config.  Indeed, backups folder is empty, saved folder is empty.  Weird.  Nothing in the trash can on the synology.

Any ideas?  Is the planet lost to the multiverse?

**Your Runtime Command or Docker Compose File**
[wolveix-satisfactory-server-1.syno.json](https://github.com/wolveix/satisfactory-server/files/13690002/wolveix-satisfactory-server-1.syno.json)

**System Specs (please complete the following information):**
running on synology 1821+ via Docker, dsm 7.2.1

```
echo "===== START ISSUE REPORT =====
OS:  $(uname -a)
CPU: $(lscpu | grep 'Model name:' | sed 's/Model name:[[:space:]]*//g')
RAM: $(awk '/MemAvailable/ {printf( "%d\n", $2 / 1024000 )}' /proc/meminfo)GB/$(awk '/MemTotal/ {printf( "%d\n", $2 / 1024000 )}' /proc/meminfo)GB
HDD: $(df -h | awk '$NF=="/"{printf "%dGB/%dGB (%s used)\n", $3,$2,$5}')
===== END ISSUE REPORT ====="
```

Alternatively, you can find the information manually. Here's what we're looking for:
-   OS: [e.g. Ubuntu 18.04 x86_64] (Linux: `uname -a`)
-   CPU: [e.g. AMD Ryzen 5 3600 6-Core Processor] (Linux: `lscpu`)
-   RAM: [e.g. 4GB/16GB] (Linux: `cat /proc/meminfo | grep Mem`)
-   HDD; [e.g. 22GB/251GB (9% used)] (Linux: `df -h`)

**Additional Context**

**Comment by wolveix** [comment]
Hey @egoodchild, that doesn't sound fun! My best guess would be that your volume wasn't properly mapped, causing a container update/rebuild to wipe the data.

Out of curiosity, does `/docker/satisfactory/gamefiles` contain anything on your Synology device?

**Comment by egoodchild** [comment]
Yep.  
![image](https://github.com/wolveix/satisfactory-server/assets/8422475/a5c1628a-ddc1-49c0-99d8-b6cbbb7a8a34)

**Comment by wolveix** [comment]
That's strange. The `backups` directory would be empty if the container hadn't ever been restarted. I'm unaware of any such issue causing saves to be deleted, either. Sorry I can't really be anymore help!

Did your server log have anything worthwhile in? Or have you rebuilt (not restarted, rebuilt) the container since then?

**Comment by wolveix** [comment]
The full logfile would be very useful, as it may give us an idea of what may have happened

**Comment by egoodchild** [comment]
Where do said logs normally live?  I will try to get for ya.

**Comment by wolveix** [comment]
I'm referring to the container's logs, as opposed to game server logs :) I'd assume Synology give you some kind of way to view/download the logs

**Comment by wolveix** [comment]
@egoodchild any luck with those logs? :)

**Comment by egoodchild** [comment]
I'm guessing this happened because I run watchtower and it updated stuff.  I haven't grabbed the logs yet.  Having trouble trying to push the save file folder out of the container config, so I am not sure if I can run this container while I also run watchtower.

I'll try to grab the logs now....

**Comment by wolveix** [comment]
@egoodchild is the image was updated, the logs may not be useful.

Updating the image should have zero effect on the save files, which is why I'm a little confused as to what may have happened here

**Comment by egoodchild** [comment]
Looks like I limited the log size to ~100kb so it's lost stuff before the 16th.  I once had some docker install that took down a raspberry pi because it had some unlimited log size :P 

Sharing what's here anyway.

**Comment by egoodchild** [comment]
[wolveix-satisfactory-server-1.zip](https://github.com/wolveix/satisfactory-server/files/13719715/wolveix-satisfactory-server-1.zip)

**Comment by wolveix** [comment]
Yeah, not seeing anything too useful there unfortunately. I'm afraid I'm not sure what to recommend :( Without the original logs from when the incident occurred, it's hard to say what could've happened

**Comment by wolveix** [comment]
As we can't reproduce this, I'll close this for now. I suspect some kind of environmental factor caused it, though I can't say for certain

**Comment by egoodchild** [comment]
Hey Wolveix, what is the default save folder for your software?

I am having trouble trying to redirect it or even find the save files.  Any help appreciated!

**Comment by wolveix** [comment]
@egoodchild per the README, you'll find your save files under `/config/saved`, specifically under the `server` sub-directory

**Comment by egoodchild** [comment]
On my synology at least, they don't seem to be there.  But it does have files!  I just can't find them.  

Bleuch :)
